modules/integration-tests/src/main/java/org/apache/fluo/integration/impl/StochasticBankIT.java

```diff
@@ -195,7 +193,7 @@ private static void runVerifier(Environment env, int numAccounts, int num) {
         long t1 = System.currentTimeMillis();
 
         TestTransaction tx = new TestTransaction(env);
-        Stat stat = new Stat();
+        org.apache.accumulo.core.util.Stat stat = new org.apache.accumulo.core.util.Stat();
```

Comment:
Could be done as follow on work.   Would be nice to use org.apache.commons.math3.stat.descriptive.SummaryStatistics instead

Reply:
This change was just to get the build not to fail. I agree switching to commons-math3 would be good, and that it's a follow-on thing.

Reply:
I can make that change.

Reply:
I have already made that change in my Accumulo 2.0 branch.  The methods on that class changed in Accumulo 2.0.